src/quacc/recipes/espresso/dos.py

```diff
@@ -125,6 +129,49 @@ def projwfc_job(
     )
 
 
+@job
+def bands_pp_job(
```

Comment:
It would be helpful to clarify a bit more about the distinction between `bands_job` and `bands_pp_job`. Presumably the latter is only doing post-processing (i.e. no NSCF calculation) and relies on a prior `bands_job`? If a job relies on some other job, it's worth mentioning that in the docstring, including in the `prev_dir` description. We want to make it as easy as possible for users who may not be familiar with Espresso.